deepmd/utils/convert.py

```diff
@@ -196,12 +198,19 @@ def convert_pb_to_pbtxt(pbfile: str, pbtxtfile: str):
         filename of the input graph
     pbtxtfile : str
         filename of the output graph text
+    incompat_from_v1_to_v2 : bool
+        model_attr/model_version of TF incompatible when convert from TF1.x to TF2.x
```

Comment:
What is the usage of `incompat_from_v1_to_v2`?

Reply:
To switch on 
```
        with tf.compat.v1.gfile.GFile(pbfile, 'rb') as f:
            graph_def = tf.compat.v1.GraphDef()
```
only when convert from TF1.x to TF2.x. Please refer to rows 204-215 in `deepmd/utils/convert.py`